Wikipedia/Code/ArticleLocationCollectionViewController.swift

```diff
@@ -7,7 +7,7 @@ class ArticleLocationCollectionViewController: ColumnarCollectionViewController,
         }
     }
     let dataStore: MWKDataStore
-    fileprivate let locationManager = WMFLocationManager.fine()
+    fileprivate let locationManager = LocationManager()
```

Comment:
I wonder if we ought to reference `LocationManagerProtocol` as much as possible now that you have the abstraction. `PlacesViewController` could be updated to use it too.